content/posts/cloudbees-core-on-kubernetes-best-practices/core-psp.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Using Kubernetes Pod Security Policies with CloudBees Core
+series: ["CloudBees Core on Kubernetes Best Practices"]
+part: 1
+author:
+  name: "Kurt Madel"
+date: 2019-08-30T05:05:15-04:00
+showDate: true
+tags: ["Kubernetes","CI","CD","Core v2","security","Pod Security Policies"]
+photo: "/posts/cloudbees-core-on-kubernetes-best-practices/bank.jpg"
+photoCaption: "Wells Fargo Bank, Market Street, San Francisco, CA<br>Photograph by Kurt Madel ©2019"
+exif: "SONY RX-100 ISO 125 10.4mm ƒ/2.2 1/100"
+draft: false
+---
+## What are Pod Security Policies?
+Although [Kubernetes Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) are still a **beta** feature of Kubernetes they are an important security feature that should not be overlooked. Pod Security Policies (PSPs) are built-in Kubernetes resources that allow you to enforce security related properties of every container in your cluster. If a container in a pod does not meet the criteria for an applicable PSP then it will not be scheduled to run.
+
+## OOTB and Documented Security Best Practices for CloudBees Core v2 on Kubernetes
```

Comment:
Shorter title and putting the `OOTB and Documented` as an explanation in the body?? Just because in the Preview it's shown in the left side of the image and shows a large title in a thin column close to the image.

Reply:
Done.